### PR TITLE
Update Visualization Types - Text View of user document

### PR DESCRIPTION
**Pull request by recca5p**

### CODE REVIEW

File: docs/user/visualization-types/images/text-view-conditional-formatting.png
Comment (hainv-ohio) [inline docs/user/visualization-types/images/text-view-conditional-formatting.png:1]: 9The new image is incorrect, it should show the ability to do conditional formatting like the old image.

**Review by hainv-ohio — CHANGES_REQUESTED** [review]

File: docs/user/visualization-types/images/text-view-conditional-formatting.png
Comment (recca5p) [reply docs/user/visualization-types/images/text-view-conditional-formatting.png:1]: I have updated the new screenshot, please re-review it

Comment (hainv-ohio) [reply docs/user/visualization-types/images/text-view-conditional-formatting.png:1]: @recca5p revenue should be in currency format, not a simple number

**Review by hainv-ohio — CHANGES_REQUESTED** [review]

File: docs/user/visualization-types/text-view.md
Comment (hainv-ohio) [inline docs/user/visualization-types/text-view.md:1]: Remove ", as explained in [this section](text-view).". in line 16. It's pointing to itself.

**Review by hainv-ohio — CHANGES_REQUESTED** [review]

File: docs/user/visualization-types/images/text-view-conditional-formatting.png
Comment (recca5p) [reply docs/user/visualization-types/images/text-view-conditional-formatting.png:1]: Updated, please re-review it

File: docs/user/visualization-types/text-view.md
Comment (recca5p) [reply docs/user/visualization-types/text-view.md:1]: Removed, please re-review it

**Review by hainv-ohio — APPROVED** [review]

**Pull request merged**